src/python/grpcio_tests/tests_py3_only/interop/xds_interop_client.py

```diff
@@ -84,7 +85,7 @@ class _StatsWatcher:
     _no_remote_peer: int
     _lock: threading.Lock
     _condition: threading.Condition
-    _metadata_keys: frozenset
+    _metadata_keys: FrozenSet[str]
```

Comment:
We should make this change separately - because we may need to backport the build cfg and sh files.